src/index.js

```diff
@@ -1,9 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+<<<<<<< HEAD
+import App from './SortByInput';
```

Comment:
hey i think you may have imported this wrong App i believe come from app.js

Reply:
I got a bit confused. I'm trying to to push changes now.